Bottom: b06335b895d98ff307a6b569fe3aa2c240db7fe9
Top:    13b19e04c90b350a4ba1e3db66eef2fb0f32ff41
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-12-23 22:27:13 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
